src/components/network-map-tab.js

```diff
@@ -80,10 +86,22 @@ export const NetworkMapTab = ({
     const mapEquipments = useSelector((state) => state.mapEquipments);
     const studyUpdatedForce = useSelector((state) => state.studyUpdated);
     const mapDataLoading = useSelector((state) => state.mapDataLoading);
+    const treeModel = useSelector(
+        (state) => state.networkModificationTreeModel
+    );
+
+    const rootNodeId = useMemo(() => {
+        const rootNode = treeModel.treeNodes.find(
```

Comment:
Potentially, state.networkModificationTreeModel could be null (when we close the study), so technically that would be better.
```suggestion
        const rootNode = treeModel?.treeNodes.find(
```

Reply:
both solutions does exist in the project
I'm fixing it the way you ask me to do